pkg/controllers/scheduling/schedule.go

```diff
@@ -204,6 +221,25 @@ func selectClusters(placement *clusterapiv1alpha1.Placement, clusters []*cluster
 	return decisions
 }
 
+// Get the placement's prioritizer weights as a map.
+func getPlacementPrioritizerWeights(placement *clusterapiv1alpha1.Placement) map[string]int32 {
```

Comment:
I would combine this with getWeight, like

```go
weight := make(map[string]int32)
// copy the default weight
for name, weight := range s.prioritizerWeights {
    weight[name] = weight
}

// override the default weight
for _, pc := range placement.Spec.PrioritizerConfigs {
		weight[pc.Name] = pc.Weight
}

```

Reply:
Thanks, update the code.